core/modules/dashboard/includes/block.update.inc

```diff
@@ -59,56 +59,123 @@ class DashboardUpdateBlock extends Block {
       return;
     }
 
-    $items = array();
+    $header = array(
+      array('data' => t('Project')),
+      array('data' => t('Type')),
+      array('data' => t('Update')),
+    );
+    $updates = array();
+
     $available = update_get_available();
     module_load_include('inc', 'update', 'update.compare');
```

Comment:
`$available` can be empty for several reasons, in that case most of the following code is futile (and causes "undefined something" notices).

Reply:
You could pull the $panel up here and return it here early (with appropriate content like "dunno" or "nothing found", possibly a "refresh info" button). -- before you're doing the more expensive stuff.

Reply:
I cannot reproduce the issue any more. I'm suspecting that this has fixed it: [`b40cb52` (#3451)](https://github.com/backdrop/backdrop/pull/3451/commits/b40cb52ee48943df35379edb4b0e5df22f91a9e8)